src/NUnitFramework/framework/Internal/Filters/CompositeFilter.cs

```diff
@@ -64,6 +64,24 @@ public void Add(ITestFilter filter)
         public IList<ITestFilter> Filters { get; private set; }
 
         /// <summary>
+        /// Checks whether the CompositeFilter is matched by a test.
+        /// </summary>
+        /// <param name="test">The test to be matched</param>
+        public abstract override bool Pass(ITest test);
```

Comment:
I think that's a matter of preference. I personally prefer it even in purely internal code to force implementers to override methods that doesn't make sense how they are implemented in the base class.

But I can drop that from the commit, that's not a big deal :smile:.